sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusException.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.amqp.exception.AmqpException;
+
+/**
+ * Defines {@link ServiceBusException} which has addition properties. You can {@link ServiceBusErrorSource} to
+ * determine source of error.
+ *
+ * @see ServiceBusErrorSource
+ */
+public class ServiceBusException extends AmqpException {
```

Comment:
There's an administration aspect to SB as well and we will want to differentiate between an exception coming from the HTTP admin client vs the exception coming from AMQP. So, we should consider naming this exception accordingly.

Reply:
`ServiceBusAmqpException` ? : Putting  `Amqp`   in class name, is telling normal user complexity of protocol which they only need to know if they are dealing with amqp message. I still think it will be fine with `ServiceBusException` 

`ServiceBusAdministrationException`  could be another name for admin, when we need to do it.

Reply:
Renamed.